carbondoomsday/email_signature/utils.py

```diff
@@ -0,0 +1,60 @@
+# Inspiration from:
+# https://code-maven.com/create-images-with-python-pil-pillow
+# https://stackoverflow.com/a/38629258/2532070
+# https://stackoverflow.com/a/273962/2532070
+# https://stackoverflow.com/a/2563883/2532070
+# add_border_radius method:
+# https://stackoverflow.com/a/11291419/2532070
```

Comment:
If these are useful to help understand the following code then I'd rather see them closer to the code that is using something from them. That might be in a function docstring in the usual `""" my docstring """`? If more general, could also use the module docstring at the top here.